pkg/restapi/operation/operations.go

```diff
@@ -83,18 +96,42 @@ func (o *Operation) createKeystoreHandler(rw http.ResponseWriter, req *http.Requ
 	rw.WriteHeader(http.StatusCreated)
 }
 
-func (o *Operation) createKeystore() (string, error) {
-	guid := uuid.New()
-	keystoreID := fmt.Sprintf(keystoreIDFormat, guid)
+func (o *Operation) createKeystore(config keystore.Configuration) (string, error) {
+	keystoreID := buildKeystoreID()
 
 	err := o.keystore.CreateStore(keystoreID)
+
+	if err == storage.ErrDuplicateStore {
```

Comment:
better to use `errors.Is(err, storage.ErrDuplicateStore)`

Reply:
fixed